kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/main/java/org/kie/workbench/common/stunner/client/widgets/explorer/tree/TreeExplorerView.java

```diff
@@ -16,22 +16,22 @@
 
 package org.kie.workbench.common.stunner.client.widgets.explorer.tree;
 
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 import com.google.gwt.core.client.GWT;
 import com.google.gwt.dom.client.Style;
 import com.google.gwt.uibinder.client.UiBinder;
 import com.google.gwt.uibinder.client.UiField;
 import com.google.gwt.user.client.ui.Composite;
 import com.google.gwt.user.client.ui.IsWidget;
-import com.google.gwt.user.client.ui.Tree;
-import com.google.gwt.user.client.ui.TreeItem;
 import com.google.gwt.user.client.ui.Widget;
+import org.uberfire.ext.widgets.core.client.tree.Tree;
+import org.uberfire.ext.widgets.core.client.tree.TreeItem;
 
 public class TreeExplorerView extends Composite implements TreeExplorer.View {
 
-    interface ViewBinder extends UiBinder<Widget, TreeExplorerView> {
-
-    }
-
+    private static Logger LOGGER = Logger.getLogger(TreeExplorerView.class.getName());
```

Comment:
Why not adding generic on this `Tree` instance (ex: `Tree<TreeItem>`)? The IDE complains too much about generics isnt it.. :)